packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx

```diff
@@ -129,7 +129,7 @@ export const VizTooltipRow = ({
   return (
     <div className={styles.contentWrapper}>
       {(color || label) && (
-        <div className={styles.valueWrapper}>
```

Comment:
This is forcing the label and quantity fields to flex equally, which results in a lot of whitespace next to the quantity and a much shorter label field.